# Read the dataset

In [16]:
import pandas as pd

dataFrame = pd.read_pickle(r'../data/7587_corrige.pkl')
dataFrame.head()
#print(dataFrame.label.to_string())
#print(dataFrame.dtypes)

companyName  \
id                                                               
548398365_0                                            TEHTRIS   
548398365_1                                  CHEOPS TECHNOLOGY   
551303934_0                                              ALTEN   
551303934_1                                    FRANCE SECURITE   
551303934_2  OASU (Observatoire Aquitain des Sciences de l'...   

                                                    companyUrl  \
id                                                               
548398365_0          https://www.linkedin.com/company/tehtris/   
548398365_1  https://www.linkedin.com/company/cheops-techno...   
551303934_0            https://www.linkedin.com/company/alten/   
551303934_1  https://www.linkedin.com/company/france-securite/   
551303934_2                                               None   

                                                      jobTitle  \
id                                                               
548398365_0                       R&D Cybersecurity Consultant   
548398365_1                           Ingénieur en data mining   
551303934_0  Stagiaire ingénieur en intelligence artificiel...   
551303934_1                Stagiaire en développement logiciel   
551303934_2                     Stagiaire en développement Web   

                            dateRange                              location  \
id                                                                            
548398365_0   oct. 2021 – Aujourd’hui  Bordeaux, Nouvelle-Aquitaine, France   
548398365_1   sept. 2020 – sept. 2021  Bordeaux, Nouvelle-Aquitaine, France   
551303934_0  juil. 2021 – Aujourd’hui              Rennes, Bretagne, France   
551303934_1     juin 2019 – août 2019           Bordeaux, Aquitaine, France   
551303934_2     avr. 2018 – juin 2018           Bordeaux, Aquitaine, France   

                                                   description  \
id                                                               
548398365_0                                               None   
548398365_1                                               None   
551303934_0  Conception et programmation pour une plateform...   
551303934_1  Développement et mises à jour d’applications c...   
551303934_2  Création et évolution d’une application Web d’...   

                                                       logoUrl  label  
id                                                                     
548398365_0  https://media-exp1.licdn.com/dms/image/C4D0BAQ...    NaN  
548398365_1  https://media-exp1.licdn.com/dms/image/C560BAQ...    2.0  
551303934_0  https://media-exp1.licdn.com/dms/image/C4D0BAQ...    4.0  
551303934_1  https://media-exp1.licdn.com/dms/image/C4D0BAQ...    2.0  
551303934_2                                                       1.0

This dataset is in french.
All fields are strings except for *label*, which is a float number.

- **id** : Work experience ID (unique) in the format *profileID_numWorkExperience* (e.g. 548398365_0: latest work experience of the user whose ID is 548398365).
- **companyName** : Company where the user has worked.
- **companyUrl** : URL of the LinkedIn page of the company.
- **jobTitle** : Work experience title.
- **dataRange** : Date range of the work experience in the format: *startDate - endDate · duration* or *date · duration* (if startDate = endDate) (e.g. sept. 2020 - déc. 2020 · 4 mois)
- **location** : Location of the work experience. Here there are only profiles with at least one work experience in one of the 5 cities in France : Bordeaux, Nice, Lille, Lyon and Strasbourg.
- **description** : Work experience description (under the job title of an experience on the user's profile)
- **logoUrl** : **This variable must be ignored**
- **label** : Work experience are labelized from 0 to 4.
	- **0**: This experience is not related to AI
	- **1**: It's unlikely that it's related to it
	- **2**: We don't know wether it's related or not to it
	- **3**: It's likely that it's related to it
	- **4**: It's related to AI
	- **NaN**: It's often a work experience that should be labelized between 0 and 2, so it's not a work experience related to AI